parm/soca/obs/config/insitu_salt_profile_argo.yaml

```diff
@@ -0,0 +1,386 @@
+obs space:
+  name: insitu_salt_profile_argo
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: ${DATA}/obs/${OPREFIX}insitu_salt_profile_argo.${PDY}${cyc}.nc4
+    obsgrouping:
+      group variables: [latitude, longitude, dateTime]
+      sort variable: depth
+      sort group: MetaData
+      sort order: ascending
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: ${DATA}/diags/insitu_salt_profile_argo.${PDY}${cyc}.nc4
+      # simulated variables: [waterTemperature, salinity]
+      # observed variables: [waterTemperature, salinity]
```

Comment:
Do we want to leave these two line commented out?

Reply:
Yes, or even better, delete!
@AndrewEichmann-NOAA , can you clean-up the yamls that you added and remove the commented out sections.